jobs/jobs_api_go_sdk.go

```diff
@@ -313,6 +313,9 @@ func Read(jobID int64, w *databricks.WorkspaceClient, ctx context.Context) (job
 		JobId: jobID,
 	})
 	err = wrapMissingJobError(err, fmt.Sprintf("%d", jobID))
+	if err != nil {
+		return
+	}
```

Comment:
Should we add a test for this? i.e before it panics and now it works?

Reply:
My understanding is that the panic happens when the job is removed on backend and .Get return nil?
> backend changed the response from zeroed struct to nil, or sdk decodes it differently

If so we should be able to return nil from the test fixtures?